docs/contributing.rst

```diff
@@ -82,6 +82,12 @@ To run a basic set of tests that neither require ``aleth`` nor ``libz3``, run
 ``./scripts/soltest.sh --no-ipc --no-smt``. This script will run ``build/test/soltest``
 internally.
 
+.. note ::
+
+    Those working in a Windows environment wanting to run the above basic sets without aleth or libz3 in Git Bash, you would have to do
+    a different command: ``./build/test/RelWithDebInfo/soltest.exe -- --no-ipc --no-smt``. Make sure you followed the `building from source <https://solidity.readthedocs.io/en/latest/installing-solidity.html#building-from-source>`_
+    and that your Cmake was successful. If you're running this in plain Windows Command line, use ``.\build\test\RelWithDebInfo\soltest.exe -- --no-ipc --no-smt``. 
```

Comment:
This also applies to non-Windows development platforms. `soltest.sh` MAY also become superfluous since at least the `--testpath` logic was just lately merged into the soltest program itself, not requiring `soltest.sh` to do that magic anymore.

What *is* different on Windows however is how to access the aleth's IPC socket.

Reply:
soltest is still more convenient, since you do not have to care about splitting into boost and non-boost options and can use the `--debug` switch.

Reply:
@chriseth @christianparpart Should I keep the run test command or should I change up some flags?